docker_integration_test.go

```diff
@@ -12,7 +12,7 @@ import (
 
 	"github.com/garyburd/redigo/redis"
 	"github.com/mattbaird/elastigo/lib"
-	. "github.com/ory-am/dockertest"
+	. "github.com/miko-code/dockertest"
```

Comment:
This is why code coverage decreased.

This is how I do it:

1. clone the original repository to my GOPATH, e.g. `$GOPATH/src/github.com/miko-code/dockertest`
2. add a git remote to my fork, e.g. `git remote add my-fork git:/...`
3. fetch and check out the fork

this will save you the troubles of having multiple directories for the same code and avoid this issue